stacks/CognitoStack.ts

```diff
@@ -28,6 +28,7 @@ export const CognitoStack = ({ stack }: StackContext) => {
         timeout: 30,
         memorySize: 256,
         runtime: "python3.9",
+        enableLiveDev: false,
```

Comment:
why is this required?

Reply:
The admin create user command was timing out and expected the stack to be up locally